tests/static/app_registry.yaml

```diff
@@ -82,3 +82,5 @@ releases:
             title: Quantum ESPRESSO
             version: 23.1.0b1
         url: git+https://github.com/aiidalab/aiidalab-qe.git@d89b58390c0a691bf16dfaa50036bc290fca017c
+    invalid_version:
+        url: git+https://github.com/aiidalab/aiidalab-qe.git@d89b58390c0a691bf16dfaa50036bc290fca017c
```

Comment:
How does the test work? You added an `invalid_version` here, so later, the test should check that the `invalid_version` does not exist in the releases.

Reply:
Good point. It was tested implicitly, since the new `packaging` versions would raise an error. 
I've added an explicit check in `tests/test_appclass.py`.